addons/point_of_sale/static/src/js/Screens/PaymentScreen/PaymentScreen.js

```diff
@@ -189,7 +189,7 @@ odoo.define('point_of_sale.PaymentScreen', function (require) {
             }
         }
         async _finalizeValidation() {
-            if ((this.currentOrder.is_paid_with_cash() || this.currentOrder.get_change()) && this.env.pos.config.iface_cashdrawer && this.env.pos.config.use_proxy) {
+            if ((this.currentOrder.is_paid_with_cash() || this.currentOrder.get_change()) && this.env.pos.config.iface_cashdrawer && this.env.proxy.printer) {
```

Comment:
I believe `proxy` may not be present.

```suggestion
            if ((this.currentOrder.is_paid_with_cash() || this.currentOrder.get_change()) && this.env.pos.config.iface_cashdrawer && this.env.proxy && this.env.proxy.printer) {
```

Reply:
in my case (and the case that the other PR described) it was defined. But I think it's worth to double check anyway :)